torch/_subclasses/fake_tensor.py

```diff
@@ -869,12 +859,6 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
             with self:
                 return func.prim_meta_impl(*args, **kwargs)
 
-        if has_symbolic_sizes:
```

Comment:
why are we getting rid of this?

Reply:
Well, I'm trying to unify symbolic and non-symbolic behavior. There is a major downside to dumping this, which is that the error messages are dramatically less useful here (now instead of erroring here, we'll go off into the C++ and hit an error there)